index.html

```diff
@@ -3500,18 +3499,129 @@ <h3>
 Definition Language (XSD) 1.1 Part 2: Datatypes</a> [[XMLSCHEMA11-2]]. This
 datetime value MUST be normalized to UTC 00:00, as indicated by the trailing
 "Z".
-                </dd>
-                <dt>
-updated
-                </dt>
-                <dd>
+</p>
+            </section>
+
+            <section>
+                <h4>
+                  updated
+                </h4>
+                <p>
 DID document metadata SHOULD include an <code>updated</code> property
 to indicate the timestamp of the last <a href="#update">Update operation</a>.
 This property MAY not be supported by a given <a>DID method</a>.
 The value of the property MUST follow the same formatting rules as the
 <code>created</code> property.
-                </dd>
-            </dl>
+                </p>
+              </section>
+
+            <section>
+              <h4>equivalentId</h4>
+                <p>
+                  A <a>DID Method</a> can define different forms of a DID that are 
+                  logically equivalent. An example is when a DID takes one form prior 
+                  to registration in a verifiable data registry and another form after 
+                  such registration. In this case, the <a>DID Method</a> specification 
+                  may need to express one or more DIDs that are logically equivalent to 
+                  the resolved DID as a property of the DID document. This is the 
+                  purpose of the <code><a>equivalentId</a></code>  property.
+                </p>
+        
+                <dl>
+                  <dt><dfn>equivalentId</dfn></dt>
+                  <dd>
+                    The value of <code>equivalentId</code> MUST be a 
+                    <a data-cite="INFRA#lists">list</a> where each item in the list 
+                    is a DID conforming to the requirements in Section 3.1.
```

Comment:
```suggestion
                    is a <a data-cite="INFRA#string">string</a> that conforms to the rules in Section <a href="#did-syntax"></a>
```

Reply:
Suggested to line up the text with the definition supplied for the `id` property of the did document